### PR TITLE
Change `batch_norm` vectorization policy

### PR DESCRIPTION
**Pull request by xytintel**
We don’t need to vectorize FP32 currently.

### CODE REVIEW

**Comment by xytintel** [comment]
> What's the motivation? Why does FP32 not require vectorization?

The optimized batch normalization has an accuracy issue in CosmicTagger's long train convergence (FP32), and the cause is still unclear. We will disable it for now.

**Review by chunhuanMeng — APPROVED** [review]

**Comment by fengyuan14** [comment]
> > What's the motivation? Why does FP32 not require vectorization?
> 
> The optimized batch normalization has an accuracy issue in CosmicTagger's long train convergence (FP32), and the cause is still unclear. We will disable it for now.

Are you sure of if the changes leaded to the accuracy regression?
